django_afip/models.py

```diff
@@ -1493,6 +1493,13 @@ class ReceiptEntry(models.Model):
         decimal_places=2,
         help_text=_("Price per unit before vat or taxes."),
     )
+    discount = models.DecimalField(
+        _("discount"),
+        max_digits=15,
+        decimal_places=2,
+        default=0,
+        help_text=_("Total net discount applied to tax base."),
```

Comment:
We can include a validator here to make sure it's positive:

```suggestion
        help_text=_("Total net discount applied to tax base."),
        validators=[MinValueValidator(Decimal("0"))]
```

The relevant import is:

```py
from django.core.validators import MinValueValidator
```